src/screens/settings-test.tsx

```diff
@@ -391,9 +391,9 @@ export const SettingsTestScreen = observer(() => {
     const unsignedTx = await EthNetwork.populateTransaction(
       wallet.address,
       contract,
-      new BN(100000000000000),
+      new Balance(100000000000000, 0),
       data,
-      250000,
+      new Balance(250000, 0),
```

Comment:
What is 0 precision and why?

Reply:
not to write new Balance(0.00000000000025)